docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/integrate-your-charm-with-postgresql.md

```diff
@@ -199,7 +199,7 @@ Now, let's define this property such that, every time it is called, it dynamical
 
 ```python
 @property
-def app_environment(self) -> dict[str, str | None]:
+def app_environment(self) -> Dict[str, str]:
```

Comment:
We made the `Dict` to `dict` change deliberately a while back, to make for tidier code and assume that anyone doing the tutorial is using more modern Python. Could we keep that, or does it cause issues with the testing?

Reply:
Changed back to `dict`.